lib/manager/components/transform/CustomCSVForm.js

```diff
@@ -47,15 +63,29 @@ const CustomCSVForm = (props: Props) => {
           value={csvData} />
       </label>
       <div style={{marginBottom: '10px'}}>
-        <Button
+        <input
```

Comment:
I was also more thinking of the file input (for transformations) as a separate input mode, not one that would populate the CSV text area, since one of the weaknesses of that text area is it can't display very large files (such as those for stop_times in a custom field preservation transformation). Can we make the file input a separate input mode for transformations and remove the population behaviour there?

Reply:
Should we throw this to our weekly discussion call? I've tested this with large files and it is performant, and the feedback is helpful

Reply:
Sure we can discuss on the call. I can send a feed that usually shows slow input into the csv text area as well